src/test/java/com/microsoft/util/ElementUtilTest.java

```diff
@@ -60,8 +60,8 @@ public void extractSortedElements() {
 
         // Ensure items to exclude exist.
         assertThat("Wrong enclosed elements number", allElements.size(), is(6));
-        assertTrue("Unexpected package private class", allElements.contains("com.microsoft.samples.subpackage.InternalException"));
-        assertTrue("Unexpected to-exclude class", allElements.contains("com.microsoft.samples.subpackage.SomeExcludedClass"));
+        assertThat("Unexpected package private class", allElements.contains("com.microsoft.samples.subpackage.InternalException"), is(true));
+        assertThat("Unexpected to-exclude class", allElements.contains("com.microsoft.samples.subpackage.SomeExcludedClass"),is(true));

```

Comment:
Please consolidate code spacing format. `), is(true)` or `),is(true)`, choose one.

Reply:
Thanks, corrected